<summary>

https://pipelines.actions.githubusercontent.com/serviceHosts/bf0e08c1-97a1-4471-a81d-dfeb4c91dd29/_apis/pipelines/1/runs/7/signedlogcontent/2?urlExpires=2022-09-20T06%3A25%3A32.8006631Z&urlSigningMethod=HMACV1&urlSignature=9QFmanFxa0vIgV5PRTT7POeYfnMvaFS6uFvWPcUONaQ%3D

<details>

```sh
2022-09-20T06:23:43.0978519Z Requested labels: ubuntu-latest
2022-09-20T06:23:43.0978567Z Job defined at: tamakiii-sandbox/hello-github-summary/.github/workflows/docker-publish.yaml@refs/heads/init
2022-09-20T06:23:43.0978586Z Waiting for a runner to pick up this job...
2022-09-20T06:23:43.5010393Z Job is waiting for a hosted runner to come online.
2022-09-20T06:23:47.6452415Z Job is about to start running on the hosted runner: Hosted Agent (hosted)
2022-09-20T06:23:49.3579079Z Current runner version: '2.296.2'
2022-09-20T06:23:49.3602911Z ##[group]Operating System
2022-09-20T06:23:49.3603359Z Ubuntu
2022-09-20T06:23:49.3603639Z 20.04.5
2022-09-20T06:23:49.3603985Z LTS
2022-09-20T06:23:49.3604208Z ##[endgroup]
2022-09-20T06:23:49.3604495Z ##[group]Runner Image
2022-09-20T06:23:49.3604830Z Image: ubuntu-20.04
2022-09-20T06:23:49.3605153Z Version: 20220905.1
2022-09-20T06:23:49.3605569Z Included Software: https://github.com/actions/runner-images/blob/ubuntu20/20220905.1/images/linux/Ubuntu2004-Readme.md
2022-09-20T06:23:49.3606143Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu20%2F20220905.1
2022-09-20T06:23:49.3606551Z ##[endgroup]
2022-09-20T06:23:49.3606822Z ##[group]Runner Image Provisioner
2022-09-20T06:23:49.3607168Z 1.0.0.0-main-20220907-1
2022-09-20T06:23:49.3607455Z ##[endgroup]
2022-09-20T06:23:49.3608021Z ##[group]GITHUB_TOKEN Permissions
2022-09-20T06:23:49.3608501Z Contents: read
2022-09-20T06:23:49.3608961Z Metadata: read
2022-09-20T06:23:49.3609278Z Packages: write
2022-09-20T06:23:49.3609577Z ##[endgroup]
2022-09-20T06:23:49.3612737Z Secret source: Actions
2022-09-20T06:23:49.3613244Z Prepare workflow directory
2022-09-20T06:23:49.4351514Z Prepare all required actions
2022-09-20T06:23:49.4515904Z Getting action download info
2022-09-20T06:23:49.6399813Z Download action repository 'actions/checkout@v3' (SHA:2541b1294d2704b0964813337f33b291d3f8596b)
2022-09-20T06:23:49.9558398Z Download action repository 'docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9' (SHA:f054a8b539a109f9f41c372932f1ae047eff08c9)
2022-09-20T06:23:50.1286337Z Download action repository 'docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38' (SHA:98669ae865ea3cffbcbaa878cf57c20bbf1c6c38)
2022-09-20T06:23:50.3490360Z Download action repository 'docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc' (SHA:ad44023a93711e3deb337508980b4b5e9bcdc5dc)
2022-09-20T06:23:50.7606262Z ##[group]Run actions/checkout@v3
2022-09-20T06:23:50.7606561Z with:
2022-09-20T06:23:50.7606853Z   repository: tamakiii-sandbox/hello-github-summary
2022-09-20T06:23:50.7607377Z   token: ***
2022-09-20T06:23:50.7607577Z   ssh-strict: true
2022-09-20T06:23:50.7607815Z   persist-credentials: true
2022-09-20T06:23:50.7608045Z   clean: true
2022-09-20T06:23:50.7608249Z   fetch-depth: 1
2022-09-20T06:23:50.7608453Z   lfs: false
2022-09-20T06:23:50.7608653Z   submodules: false
2022-09-20T06:23:50.7608871Z   set-safe-directory: true
2022-09-20T06:23:50.7609111Z ##[endgroup]
2022-09-20T06:23:51.0219969Z Syncing repository: tamakiii-sandbox/hello-github-summary
2022-09-20T06:23:51.0221784Z ##[group]Getting Git version info
2022-09-20T06:23:51.0222407Z Working directory is '/home/runner/work/hello-github-summary/hello-github-summary'
2022-09-20T06:23:51.0222996Z [command]/usr/bin/git version
2022-09-20T06:23:51.0340395Z git version 2.37.3
2022-09-20T06:23:51.0363526Z ##[endgroup]
2022-09-20T06:23:51.0379769Z Temporarily overriding HOME='/home/runner/work/_temp/e9e16899-b35a-4100-a156-02ed58bb338e' before making global git config changes
2022-09-20T06:23:51.0380850Z Adding repository directory to the temporary git global config as a safe directory
2022-09-20T06:23:51.0385600Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/hello-github-summary/hello-github-summary
2022-09-20T06:23:51.0430063Z Deleting the contents of '/home/runner/work/hello-github-summary/hello-github-summary'
2022-09-20T06:23:51.0433642Z ##[group]Initializing the repository
2022-09-20T06:23:51.0436993Z [command]/usr/bin/git init /home/runner/work/hello-github-summary/hello-github-summary
2022-09-20T06:23:51.0498691Z hint: Using 'master' as the name for the initial branch. This default branch name
2022-09-20T06:23:51.0499467Z hint: is subject to change. To configure the initial branch name to use in all
2022-09-20T06:23:51.0500865Z hint: of your new repositories, which will suppress this warning, call:
2022-09-20T06:23:51.0501295Z hint: 
2022-09-20T06:23:51.0502107Z hint: 	git config --global init.defaultBranch <name>
2022-09-20T06:23:51.0502713Z hint: 
2022-09-20T06:23:51.0503435Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2022-09-20T06:23:51.0504017Z hint: 'development'. The just-created branch can be renamed via this command:
2022-09-20T06:23:51.0504395Z hint: 
2022-09-20T06:23:51.0504689Z hint: 	git branch -m <name>
2022-09-20T06:23:51.0517854Z Initialized empty Git repository in /home/runner/work/hello-github-summary/hello-github-summary/.git/
2022-09-20T06:23:51.0527926Z [command]/usr/bin/git remote add origin https://github.com/tamakiii-sandbox/hello-github-summary
2022-09-20T06:23:51.0577982Z ##[endgroup]
2022-09-20T06:23:51.0578618Z ##[group]Disabling automatic garbage collection
2022-09-20T06:23:51.0584357Z [command]/usr/bin/git config --local gc.auto 0
2022-09-20T06:23:51.0615607Z ##[endgroup]
2022-09-20T06:23:51.0616227Z ##[group]Setting up auth
2022-09-20T06:23:51.0625725Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-09-20T06:23:51.0659644Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-09-20T06:23:51.1045312Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-09-20T06:23:51.1066035Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-09-20T06:23:51.1285492Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2022-09-20T06:23:51.1320829Z ##[endgroup]
2022-09-20T06:23:51.1321643Z ##[group]Fetching the repository
2022-09-20T06:23:51.1329492Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +baa3c9651262f7103f22984fd31ef1c7634a9d78:refs/remotes/origin/init
2022-09-20T06:23:51.3496494Z remote: Enumerating objects: 7, done.        
2022-09-20T06:23:51.3496825Z remote: Counting objects:  14% (1/7)        
2022-09-20T06:23:51.3497117Z remote: Counting objects:  28% (2/7)        
2022-09-20T06:23:51.3497383Z remote: Counting objects:  42% (3/7)        
2022-09-20T06:23:51.3497633Z remote: Counting objects:  57% (4/7)        
2022-09-20T06:23:51.3497895Z remote: Counting objects:  71% (5/7)        
2022-09-20T06:23:51.3498166Z remote: Counting objects:  85% (6/7)        
2022-09-20T06:23:51.3498426Z remote: Counting objects: 100% (7/7)        
2022-09-20T06:23:51.3498692Z remote: Counting objects: 100% (7/7), done.        
2022-09-20T06:23:51.3498984Z remote: Compressing objects:  20% (1/5)        
2022-09-20T06:23:51.3499266Z remote: Compressing objects:  40% (2/5)        
2022-09-20T06:23:51.3499526Z remote: Compressing objects:  60% (3/5)        
2022-09-20T06:23:51.3499817Z remote: Compressing objects:  80% (4/5)        
2022-09-20T06:23:51.3500093Z remote: Compressing objects: 100% (5/5)        
2022-09-20T06:23:51.3500379Z remote: Compressing objects: 100% (5/5), done.        
2022-09-20T06:23:51.3501034Z remote: Total 7 (delta 0), reused 4 (delta 0), pack-reused 0        
2022-09-20T06:23:51.3563483Z From https://github.com/tamakiii-sandbox/hello-github-summary
2022-09-20T06:23:51.3564253Z  * [new ref]         baa3c9651262f7103f22984fd31ef1c7634a9d78 -> origin/init
2022-09-20T06:23:51.3591234Z ##[endgroup]
2022-09-20T06:23:51.3595875Z ##[group]Determining the checkout info
2022-09-20T06:23:51.3596698Z ##[endgroup]
2022-09-20T06:23:51.3597243Z ##[group]Checking out the ref
2022-09-20T06:23:51.3599488Z [command]/usr/bin/git checkout --progress --force -B init refs/remotes/origin/init
2022-09-20T06:23:51.3650209Z Switched to a new branch 'init'
2022-09-20T06:23:51.3654589Z branch 'init' set up to track 'origin/init'.
2022-09-20T06:23:51.3657011Z ##[endgroup]
2022-09-20T06:23:51.3774344Z [command]/usr/bin/git log -1 --format='%H'
2022-09-20T06:23:51.3774750Z 'baa3c9651262f7103f22984fd31ef1c7634a9d78'
2022-09-20T06:23:51.4022963Z ##[group]Run docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
2022-09-20T06:23:51.4023280Z with:
2022-09-20T06:23:51.4023462Z   registry: ghcr.io
2022-09-20T06:23:51.4023696Z   username: tamakiii
2022-09-20T06:23:51.4024199Z   password: ***
2022-09-20T06:23:51.4024390Z   logout: true
2022-09-20T06:23:51.4024588Z ##[endgroup]
2022-09-20T06:23:51.5461393Z Logging into ghcr.io...
2022-09-20T06:23:54.2622352Z Login Succeeded!
2022-09-20T06:23:54.2756389Z ##[group]Run docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
2022-09-20T06:23:54.2756711Z with:
2022-09-20T06:23:54.2756977Z   images: ghcr.io/tamakiii-sandbox/hello-github-summary

2022-09-20T06:23:54.2757450Z   github-token: ***
2022-09-20T06:23:54.2757667Z ##[endgroup]
2022-09-20T06:23:54.5749904Z ##[group]Context info
2022-09-20T06:23:54.5750536Z eventName: push
2022-09-20T06:23:54.5751300Z sha: baa3c9651262f7103f22984fd31ef1c7634a9d78
2022-09-20T06:23:54.5751546Z ref: refs/heads/init
2022-09-20T06:23:54.5751781Z workflow: Publish Docker image
2022-09-20T06:23:54.5751983Z action: meta
2022-09-20T06:23:54.5752156Z actor: tamakiii
2022-09-20T06:23:54.5752335Z runNumber: 1
2022-09-20T06:23:54.5753571Z runId: 3087871285
2022-09-20T06:23:54.5754206Z ##[endgroup]
2022-09-20T06:23:54.5766826Z ##[group]Processing tags input
2022-09-20T06:23:54.5767530Z type=schedule,pattern=nightly,enable=true,priority=1000
2022-09-20T06:23:54.5768193Z type=ref,event=branch,enable=true,priority=600
2022-09-20T06:23:54.5768768Z type=ref,event=tag,enable=true,priority=600
2022-09-20T06:23:54.5769843Z type=ref,event=pr,prefix=pr-,enable=true,priority=600
2022-09-20T06:23:54.5770581Z ##[endgroup]
2022-09-20T06:23:54.5771218Z ##[group]Processing flavor input
2022-09-20T06:23:54.5802586Z latest=auto
2022-09-20T06:23:54.5803450Z prefix=
2022-09-20T06:23:54.5803658Z suffix=
2022-09-20T06:23:54.5804143Z ##[endgroup]
2022-09-20T06:23:54.5804485Z ##[group]Docker image version
2022-09-20T06:23:54.5804925Z init
2022-09-20T06:23:54.5805600Z ##[endgroup]
2022-09-20T06:23:54.5827909Z ##[group]Docker tags
2022-09-20T06:23:54.5828380Z ghcr.io/tamakiii-sandbox/hello-github-summary:init
2022-09-20T06:23:54.5828825Z ##[endgroup]
2022-09-20T06:23:54.5829721Z ##[group]Docker labels
2022-09-20T06:23:54.5830091Z org.opencontainers.image.title=hello-github-summary
2022-09-20T06:23:54.5830429Z org.opencontainers.image.description=
2022-09-20T06:23:54.5830938Z org.opencontainers.image.url=https://github.com/tamakiii-sandbox/hello-github-summary
2022-09-20T06:23:54.5831522Z org.opencontainers.image.source=https://github.com/tamakiii-sandbox/hello-github-summary
2022-09-20T06:23:54.5831909Z org.opencontainers.image.version=init
2022-09-20T06:23:54.5832339Z org.opencontainers.image.created=2022-09-20T06:23:54.576Z
2022-09-20T06:23:54.5832703Z org.opencontainers.image.revision=baa3c9651262f7103f22984fd31ef1c7634a9d78
2022-09-20T06:23:54.5833055Z org.opencontainers.image.licenses=
2022-09-20T06:23:54.5833461Z ##[endgroup]
2022-09-20T06:23:54.5834433Z ##[group]JSON output
2022-09-20T06:23:54.5834640Z {
2022-09-20T06:23:54.5834822Z   "tags": [
2022-09-20T06:23:54.5836702Z     "ghcr.io/tamakiii-sandbox/hello-github-summary:init"
2022-09-20T06:23:54.5836979Z   ],
2022-09-20T06:23:54.5837168Z   "labels": {
2022-09-20T06:23:54.5838333Z     "org.opencontainers.image.title": "hello-github-summary",
2022-09-20T06:23:54.5838693Z     "org.opencontainers.image.description": "",
2022-09-20T06:23:54.5839227Z     "org.opencontainers.image.url": "https://github.com/tamakiii-sandbox/hello-github-summary",
2022-09-20T06:23:54.5839812Z     "org.opencontainers.image.source": "https://github.com/tamakiii-sandbox/hello-github-summary",
2022-09-20T06:23:54.5840222Z     "org.opencontainers.image.version": "init",
2022-09-20T06:23:54.5840654Z     "org.opencontainers.image.created": "2022-09-20T06:23:54.576Z",
2022-09-20T06:23:54.5841045Z     "org.opencontainers.image.revision": "baa3c9651262f7103f22984fd31ef1c7634a9d78",
2022-09-20T06:23:54.5841399Z     "org.opencontainers.image.licenses": ""
2022-09-20T06:23:54.5841792Z   }
2022-09-20T06:23:54.5842073Z }
2022-09-20T06:23:54.5842444Z ##[endgroup]
2022-09-20T06:23:54.5843417Z ##[group]Bake definition file
2022-09-20T06:23:54.5843615Z {
2022-09-20T06:23:54.5843795Z   "target": {
2022-09-20T06:23:54.5844072Z     "docker-metadata-action": {
2022-09-20T06:23:54.5844284Z       "tags": [
2022-09-20T06:23:54.5844628Z         "ghcr.io/tamakiii-sandbox/hello-github-summary:init"
2022-09-20T06:23:54.5845031Z       ],
2022-09-20T06:23:54.5845211Z       "labels": {
2022-09-20T06:23:54.5845588Z         "org.opencontainers.image.title": "hello-github-summary",
2022-09-20T06:23:54.5845950Z         "org.opencontainers.image.description": "",
2022-09-20T06:23:54.5846477Z         "org.opencontainers.image.url": "https://github.com/tamakiii-sandbox/hello-github-summary",
2022-09-20T06:23:54.5847077Z         "org.opencontainers.image.source": "https://github.com/tamakiii-sandbox/hello-github-summary",
2022-09-20T06:23:54.5847492Z         "org.opencontainers.image.version": "init",
2022-09-20T06:23:54.5847936Z         "org.opencontainers.image.created": "2022-09-20T06:23:54.576Z",
2022-09-20T06:23:54.5848323Z         "org.opencontainers.image.revision": "baa3c9651262f7103f22984fd31ef1c7634a9d78",
2022-09-20T06:23:54.5848691Z         "org.opencontainers.image.licenses": ""
2022-09-20T06:23:54.5848938Z       },
2022-09-20T06:23:54.5849107Z       "args": {
2022-09-20T06:23:54.5849500Z         "DOCKER_META_IMAGES": "ghcr.io/tamakiii-sandbox/hello-github-summary",
2022-09-20T06:23:54.5849813Z         "DOCKER_META_VERSION": "init"
2022-09-20T06:23:54.5850026Z       }
2022-09-20T06:23:54.5850184Z     }
2022-09-20T06:23:54.5850352Z   }
2022-09-20T06:23:54.5850518Z }
2022-09-20T06:23:54.5850875Z ##[endgroup]
2022-09-20T06:23:54.5903319Z ##[group]Run docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
2022-09-20T06:23:54.5903701Z with:
2022-09-20T06:23:54.5903910Z   context: .
2022-09-20T06:23:54.5904213Z   push: true
2022-09-20T06:23:54.5904574Z   tags: ghcr.io/tamakiii-sandbox/hello-github-summary:init
2022-09-20T06:23:54.5905705Z   labels: org.opencontainers.image.title=hello-github-summary
org.opencontainers.image.description=
org.opencontainers.image.url=https://github.com/tamakiii-sandbox/hello-github-summary
org.opencontainers.image.source=https://github.com/tamakiii-sandbox/hello-github-summary
org.opencontainers.image.version=init
org.opencontainers.image.created=2022-09-20T06:23:54.576Z
org.opencontainers.image.revision=baa3c9651262f7103f22984fd31ef1c7634a9d78
org.opencontainers.image.licenses=
2022-09-20T06:23:54.5906700Z   load: false
2022-09-20T06:23:54.5907013Z   no-cache: false
2022-09-20T06:23:54.5907286Z   pull: false
2022-09-20T06:23:54.5907759Z   github-token: ***
2022-09-20T06:23:54.5908096Z ##[endgroup]
2022-09-20T06:23:54.6511394Z ##[group]Docker info
2022-09-20T06:23:54.6550944Z [command]/usr/bin/docker version
2022-09-20T06:23:54.6960958Z Client:
2022-09-20T06:23:54.6964849Z  Version:           20.10.17+azure-1
2022-09-20T06:23:54.6965396Z  API version:       1.41
2022-09-20T06:23:54.6966203Z  Go version:        go1.17.11
2022-09-20T06:23:54.6966520Z  Git commit:        100c70180fde3601def79a59cc3e996aa553c9b9
2022-09-20T06:23:54.6966902Z  Built:             Mon Jun  6 21:36:39 UTC 2022
2022-09-20T06:23:54.6967266Z  OS/Arch:           linux/amd64
2022-09-20T06:23:54.6967530Z  Context:           default
2022-09-20T06:23:54.6967758Z  Experimental:      true
2022-09-20T06:23:54.6967966Z 
2022-09-20T06:23:54.6968082Z Server:
2022-09-20T06:23:54.6968354Z  Engine:
2022-09-20T06:23:54.6968733Z   Version:          20.10.17+azure-3
2022-09-20T06:23:54.6969031Z   API version:      1.41 (minimum version 1.12)
2022-09-20T06:23:54.6969329Z   Go version:       go1.17.11
2022-09-20T06:23:54.6969618Z   Git commit:       a89b84221c8560e7a3dee2a653353429e7628424
2022-09-20T06:23:54.6969954Z   Built:            Mon Jun  6 22:32:38 2022
2022-09-20T06:23:54.6970227Z   OS/Arch:          linux/amd64
2022-09-20T06:23:54.6970448Z   Experimental:     false
2022-09-20T06:23:54.6970962Z  containerd:
2022-09-20T06:23:54.6971258Z   Version:          1.5.13+azure-1
2022-09-20T06:23:54.6971556Z   GitCommit:        a17ec496a95e55601607ca50828147e8ccaeebf1
2022-09-20T06:23:54.6971836Z  runc:
2022-09-20T06:23:54.6972083Z   Version:          1.0.3
2022-09-20T06:23:54.6972396Z   GitCommit:        f46b6ba2c9314cfc8caae24a32ec5fe9ef1059fe
2022-09-20T06:23:54.6972690Z  docker-init:
2022-09-20T06:23:54.6972964Z   Version:          0.19.0
2022-09-20T06:23:54.6973221Z   GitCommit:        
2022-09-20T06:23:54.7029553Z [command]/usr/bin/docker info
2022-09-20T06:23:59.9817920Z Client:
2022-09-20T06:23:59.9818693Z  Context:    default
2022-09-20T06:23:59.9819656Z  Debug Mode: false
2022-09-20T06:23:59.9820114Z  Plugins:
2022-09-20T06:23:59.9821004Z   buildx: Docker Buildx (Docker Inc., 0.9.1+azure-1)
2022-09-20T06:23:59.9821812Z   compose: Docker Compose (Docker Inc., 2.10.2+azure-1)
2022-09-20T06:23:59.9822574Z 
2022-09-20T06:23:59.9822881Z Server:
2022-09-20T06:23:59.9823435Z  Containers: 0
2022-09-20T06:23:59.9823961Z   Running: 0
2022-09-20T06:23:59.9875785Z   Paused: 0
2022-09-20T06:23:59.9876003Z   Stopped: 0
2022-09-20T06:23:59.9876328Z  Images: 19
2022-09-20T06:23:59.9876724Z  Server Version: 20.10.17+azure-3
2022-09-20T06:23:59.9876956Z  Storage Driver: overlay2
2022-09-20T06:23:59.9877230Z   Backing Filesystem: extfs
2022-09-20T06:23:59.9877548Z   Supports d_type: true
2022-09-20T06:23:59.9877779Z   Native Overlay Diff: false
2022-09-20T06:23:59.9878043Z   userxattr: false
2022-09-20T06:23:59.9878366Z  Logging Driver: json-file
2022-09-20T06:23:59.9878594Z  Cgroup Driver: cgroupfs
2022-09-20T06:23:59.9878892Z  Cgroup Version: 1
2022-09-20T06:23:59.9879136Z  Plugins:
2022-09-20T06:23:59.9879330Z   Volume: local
2022-09-20T06:23:59.9880250Z   Network: bridge host ipvlan macvlan null overlay
2022-09-20T06:23:59.9880767Z   Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
2022-09-20T06:23:59.9881095Z  Swarm: inactive
2022-09-20T06:23:59.9881487Z  Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
2022-09-20T06:23:59.9881831Z  Default Runtime: runc
2022-09-20T06:23:59.9882302Z  Init Binary: docker-init
2022-09-20T06:23:59.9882577Z  containerd version: a17ec496a95e55601607ca50828147e8ccaeebf1
2022-09-20T06:23:59.9882966Z  runc version: f46b6ba2c9314cfc8caae24a32ec5fe9ef1059fe
2022-09-20T06:23:59.9883267Z  init version: 
2022-09-20T06:23:59.9883472Z  Security Options:
2022-09-20T06:23:59.9883732Z   apparmor
2022-09-20T06:23:59.9883987Z   seccomp
2022-09-20T06:23:59.9884177Z    Profile: default
2022-09-20T06:23:59.9884512Z  Kernel Version: 5.15.0-1019-azure
2022-09-20T06:23:59.9884807Z  Operating System: Ubuntu 20.04.5 LTS
2022-09-20T06:23:59.9885036Z  OSType: linux
2022-09-20T06:23:59.9885315Z  Architecture: x86_64
2022-09-20T06:23:59.9885567Z  CPUs: 2
2022-09-20T06:23:59.9885772Z  Total Memory: 6.781GiB
2022-09-20T06:23:59.9886080Z  Name: fv-az119-412
2022-09-20T06:23:59.9886389Z  ID: HOLX:CU2J:U55M:E64H:XZKE:2WNP:HSLH:HINJ:6G7Z:5EEX:M3S7:AG3R
2022-09-20T06:23:59.9886685Z  Docker Root Dir: /var/lib/docker
2022-09-20T06:23:59.9886944Z  Debug Mode: false
2022-09-20T06:23:59.9887212Z  Username: githubactions
2022-09-20T06:23:59.9887478Z  Registry: https://index.docker.io/v1/
2022-09-20T06:23:59.9887755Z  Labels:
2022-09-20T06:23:59.9888029Z  Experimental: false
2022-09-20T06:23:59.9888250Z  Insecure Registries:
2022-09-20T06:23:59.9888828Z   127.0.0.0/8
2022-09-20T06:23:59.9889086Z  Live Restore Enabled: false
2022-09-20T06:23:59.9889249Z 
2022-09-20T06:23:59.9889952Z ##[endgroup]
2022-09-20T06:24:00.2005047Z [command]/usr/bin/docker buildx build --label org.opencontainers.image.title=hello-github-summary --label org.opencontainers.image.description= --label org.opencontainers.image.url=https://github.com/tamakiii-sandbox/hello-github-summary --label org.opencontainers.image.source=https://github.com/tamakiii-sandbox/hello-github-summary --label org.opencontainers.image.version=init --label org.opencontainers.image.created=2022-09-20T06:23:54.576Z --label org.opencontainers.image.revision=baa3c9651262f7103f22984fd31ef1c7634a9d78 --label org.opencontainers.image.licenses= --tag ghcr.io/tamakiii-sandbox/hello-github-summary:init --iidfile /tmp/docker-build-push-z01bv1/iidfile --push .
2022-09-20T06:24:00.5397753Z #1 [internal] load build definition from Dockerfile
2022-09-20T06:24:00.5398470Z #1 transferring dockerfile: 2B done
2022-09-20T06:24:00.5399148Z #1 DONE 0.0s
2022-09-20T06:24:00.6321747Z 
2022-09-20T06:24:00.6326989Z #2 [internal] load .dockerignore
2022-09-20T06:24:00.6329883Z #2 transferring context:
2022-09-20T06:24:00.6331341Z ERROR: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount1489755853/Dockerfile: no such file or directory
2022-09-20T06:24:00.6410007Z ##[error]buildx call failed with: ERROR: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount1489755853/Dockerfile: no such file or directory
2022-09-20T06:24:00.6586029Z Post job cleanup.
2022-09-20T06:24:00.7242721Z ##[group]Removing temp folder /tmp/docker-build-push-z01bv1
2022-09-20T06:24:00.7249535Z ##[endgroup]
2022-09-20T06:24:00.7311751Z Post job cleanup.
2022-09-20T06:24:00.7744467Z [command]/usr/bin/docker logout ghcr.io
2022-09-20T06:24:00.8045446Z Removing login credentials for ghcr.io
2022-09-20T06:24:00.8167721Z Post job cleanup.
2022-09-20T06:24:00.9334746Z [command]/usr/bin/git version
2022-09-20T06:24:00.9381741Z git version 2.37.3
2022-09-20T06:24:00.9422231Z Temporarily overriding HOME='/home/runner/work/_temp/66452c8b-efcc-4d7e-b054-394dc0639c9b' before making global git config changes
2022-09-20T06:24:00.9422998Z Adding repository directory to the temporary git global config as a safe directory
2022-09-20T06:24:00.9427721Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/hello-github-summary/hello-github-summary
2022-09-20T06:24:00.9465335Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-09-20T06:24:00.9498662Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-09-20T06:24:00.9716691Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-09-20T06:24:00.9741657Z http.https://github.com/.extraheader
2022-09-20T06:24:00.9751617Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2022-09-20T06:24:00.9784044Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-09-20T06:24:01.0202813Z Cleaning up orphan processes
```

</details>

</summary>

--- a/.github/workflows/docker-publish.yaml
+++ b/.github/workflows/docker-publish.yaml
@@ -37,6 +37,9 @@ jobs:
         uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
         with:
           context: .
+          build-args: |
+            PHP_VERSION=71
+            COMPOSER_VERSION=2.2.10
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,1 +1,45 @@
 FROM amazonlinux:2.0.20220805.0
+
+ARG PHP_VERSION=81
+ARG COMPOSER_VERSION=2.4.2
+
+RUN yum update -y && \
+    yum install -y yum-utils
+
+RUN amazon-linux-extras install -y epel && \
+    yum-config-manager --enable epel && \
+    yum install -y https://rpms.remirepo.net/enterprise/remi-release-7.rpm && \
+    yum-config-manager --enable remi-php${PHP_VERSION}
+
+RUN yum update -y && \
+    yum install -y \
+      php${PHP_VERSION} \
+      php${PHP_VERSION}-php-xml \
+      php${PHP_VERSION}-php-mbstring \
+      php${PHP_VERSION}-php-process \
+      php${PHP_VERSION}-php-intl \
+      php${PHP_VERSION}-php-pdo \
+      php${PHP_VERSION}-php-opcache \
+      php${PHP_VERSION}-php-fpm \
+      make \
+      git \
+      unzip \
+      && \
+    yum clean all && \
+    rm -rf /var/cache/yum/*
+
+RUN ln -s /opt/remi/php${PHP_VERSION}/root/usr/sbin/php-fpm /usr/sbin/php${PHP_VERSION}-fpm && \
+    alternatives --install /usr/bin/php php /usr/bin/php${PHP_VERSION} 1 && \
+    alternatives --install /usr/bin/php-fpm php-fpm /usr/sbin/php${PHP_VERSION}-fpm 1
+
+RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
+    php -r "if (hash_file('sha384', 'composer-setup.php') === '55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
+    php composer-setup.php --install-dir=/usr/bin --version=${COMPOSER_VERSION} --filename=composer && \
+    php -r "unlink('composer-setup.php');"
+
+RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.rpm.sh' | bash && \
+    yum install -y symfony-cli
+
+RUN git config --global user.email "tamakiii@users.noreply.github.com" && \
+    git config --global user.name "tamakiii" && \
+    composer config --global --no-plugins allow-plugins.symfony/flex true

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,0 +1,1 @@
+FROM amazonlinux:2.0.20220805.0

